cypress/tools/fixture-generator.py

```diff
@@ -0,0 +1,54 @@
+import json
+import random
+import string
+import uuid
+from datetime import datetime, timedelta
+import os
+
+def random_string(length=8):
+    return ''.join(random.choices(string.printable + 
+                                  string.digits + 
+                                  string.punctuation + 
+                                  string.whitespace, k=length))
+
+def random_date(start, end):
+    delta = end - start
+    random_days = random.randint(0, delta.days)
+    return start + timedelta(days=random_days)
```

Comment:
@sergesoroka We're currently only testing with up to days precision. Let's tighten the random time interval to one year, e.g.:
```py
start_date = datetime(2023, 7, 1)
end_date = datetime(2024, 7, 1)
```
And let's add random microseconds with something like:
```py
random_days = random.randint(0, delta.days)
random_us = random.randint(1, 86400000000)  # intentionally have 1 as lower bound
return start + timedelta(days=random_days, microseconds=random_us)
```